modulus/sym/trainer.py

```diff
@@ -60,11 +60,21 @@ def adam_compute_gradients(
         self, aggregator: nn.Module, global_optimizer_model: nn.Module, step: int
     ):
         loss, losses = 0, Counter({})
+
+        if self.cfg.cuda_graphs and self.grad_agg_freq != 1:
+            self.log.warn(
```

Comment:
Can we please make an issue to track this?
Also, I prefer raising an exception instead of a warning and forcing `grad_agg_freq` to 1.

Reply:
Changed it to a `ValueError`.